test/test_encryption.py

```diff
@@ -62,7 +65,26 @@
                         rs_or_single_client,
                         wait_until)
 from test.utils_spec_runner import SpecRunner
-from test.test_ssl import CA_PEM
+
+try:
+    import certifi
+    HAVE_CERTIFI = True
+except ImportError:
+    HAVE_CERTIFI = False
+
+patcher = None
+
+
+def setUpModule():
```

Comment:
Is this a standard interface expected by unittest?

Reply:
Also, why is this needed if we can just call `self.patch_system_certs` wherever required?

Reply:
Yes. https://docs.python.org/3/library/unittest.html#setupmodule-and-teardownmodule

It was easier to implement this for the entire module than it was to try do it vie setUpClass() or setUp().